cmd/kubeadm/OWNERS

```diff
@@ -1,26 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- luxas
-- timothysc
```

Comment:
nit: removals from approvers should probably added to `emeritus_approvers`: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus

Reply:
thanks, i remembered about `emeritus_approvers` right after sending the PR.